x/staking/types/historical_info.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"sort"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// HistoricalInfo contains the historical information that gets stored at each height
+type HistoricalInfo struct {
+	Header abci.Header `json:"header" yaml:"header"`
```

Comment:
See the previous question. Any reason why we need the entire header and not just the minimal set of info (e.g. the height)?